.travis.yml

```diff
@@ -172,7 +172,10 @@ before_deploy:
     - docker login --email=$DOCKER_HUB_EMAIL --username=$DOCKER_HUB_USERNAME --password=$DOCKER_HUB_PASSWORD
 
     # Set latest tag only for HEAD of master branch
-    - if [ "$TRAVIS_BRANCH" = "master" ]; then docker push $DOCKER_REPOSITORY:latest; fi
+    - if [ "$TRAVIS_BRANCH" = "master" ]; then
+        docker tag $DOCKER_REPOSITORY:pr $DOCKER_REPOSITORY:latest;
```

Comment:
Because when the PR analyzer runs, if you tag it with "latest", it's a bit ambiguous. It's not "latest" until it's merged. Hence we tag it "pr" first, test it, and then once we know it passes, we retag it as latest here in the deploy section.